src/meta/src/hummock/manager/mod.rs

```diff
@@ -2519,31 +2544,69 @@ impl HummockManager {
         let default_group_id: CompactionGroupId = StaticCompactionGroupId::StateDefault.into();
         let mv_group_id: CompactionGroupId = StaticCompactionGroupId::MaterializedView.into();
         let partition_vnode_count = self.env.opts.partition_vnode_count;
+        let hybrid_vnode_count: u32 = self.env.opts.partition_vnode_count;
+        const SPLIT_BY_TABLE: u32 = 1;
+
         let window_size = HISTORY_TABLE_INFO_STATISTIC_TIME / (checkpoint_secs as usize);
+
+        let mut group_to_table_vnode_partition = HashMap::default();
```

Comment:
We will clear group_to_table_vnode_partition and re-calculate it. Just to double check: do we guarantee that `group_infos` will contain all compaction groups and all table ids?